goal-planning/goal-planning-frontend
# Chore / Add Styles to Registration Form & Add Github Workflow Templates

## Changes
1. Add CSS updates to registration form under `styles/auth-form.css`.
2. Update navbar hover highlight color to a brighter hue.
3. Added Github workflow templates for `pull request` and `issues`.

## Purpose
Add some more flare to the registration form visuals. Also, update Github workflow simplicity.

## Approach
Warp a container around the registration form that will make it look like a card or modal. Add CSS styling to make the container have a shadow effect and adjust paddings for a better layout of context.

Templates for Github workflows are attached under a folder named `.github/`. These templates are then mark down files that are saved into this folder.

<img width="1680" alt="Screenshot 2023-06-06 at 11 45 52 AM" src="https://github.com/goal-planning/goal-planning-frontend/assets/31714174/7f63195c-7fb0-40f1-885c-34a7917b74e9">

## Testing Steps
1. Pull down code
2. Run frontend application and navigate to the registration form: `http://localhost:3000/register` to view the form.
3. Go to Github.
4. Open an issue and confirm a template is pre-populated in the description.
5. Open a pull request and confirm a template is pre-populated in the description.

Closes #49
